src/explore-education-statistics-admin/src/components/editable/EditableBlockRenderer.tsx

```diff
@@ -41,6 +43,7 @@ function EditableBlockRenderer({
         <div className="dfe-content-overflow">
           <EditableBlockWrapper onDelete={editable ? handleDelete : undefined}>
             <DataBlockRenderer
+              releaseId={releaseId}
```

Comment:
Would return null on this branch of the switch statement if there isn't a `releaseId`. We shouldn't be trying to pass through an empty string.